src/react/components/icons/Info.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Tooltip } from 'oskari-ui';
+import styled from 'styled-components';
+
+
+const StyledInfoIcon = styled(InfoCircleOutlined)`
```

Comment:
Lets change this to QuestionCircleOutlined and make another icon-component that is for metadata icons. The metadata icon can use InfoCircleOutlined, but we could also bake in some logic inside it like having metadataId prop and the component render as null if metadataId is not available. The onClick for the metadataIcon can also be baked in to send the request for showing metadata (so we don't have to repeat that when using the metadata icon.